src/main/kotlin/org/jetbrains/research/ml/kotlinAnalysis/Pipeline.kt

```diff
@@ -0,0 +1,37 @@
+package org.jetbrains.research.ml.kotlinAnalysis
+
+import com.intellij.ide.impl.ProjectUtil
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Path
+
+class Pipeline(private val outputDir: Path) {
```

Comment:
Sorry, but it is a very strange name. I mean it is not clear to me what is the pipeline? Could you please rename it, also add a doc